Assets/Scripts/FengGameManagerMKII.cs

```diff
@@ -3630,8 +3639,7 @@ public void SpawnPlayerTitan()
         GameObject.Find("MainCamera").GetComponent<IN_GAME_MAIN_CAMERA>().setMainObjectASTITAN(playerTitan.gameObject);
         GameObject.Find("MainCamera").GetComponent<IN_GAME_MAIN_CAMERA>().enabled = true;
         GameObject.Find("MainCamera").GetComponent<SpectatorMovement>().disable = true;
-        //TODO MouseLook
-        //GameObject.Find("MainCamera").GetComponent<MouseLook>().disable = true;
+        GameObject.Find("MainCamera").GetComponent<Assets.Scripts.UI.Camera.MouseLook>().disable = true;
```

Comment:
Import the namespace, and use MouseLook instead of Assets.Scripts.UI.Camera.MouseLook